.github/workflows/ng-nodejs-ci.yaml

```
@@ -85,6 +85,6 @@ jobs:
       - name: Publish runified JSR
         if: startsWith(github.ref, 'refs/tags/v')
         run: |
-          cd ts/pubdir/src
-          pnpm exec deno publish --allow-dirty --unstable-sloppy-imports
+          cd ts/pubdir
+          pnpm exec deno publish --allow-dirty --unstable-sloppy-imports 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)